rules/windows/process_creation/proc_creation_win_hktl_wsass.yml

```diff
@@ -0,0 +1,31 @@
+title: HackTool - WSASS Execution for PPL Bypass and LSASS Dumping
+id: 589ac73f-8e12-409c-964e-31a2f5775ae2
+status: experimental
+description: Detects execution of WSASS, a tool used to dump LSASS memory on Windows systems by leveraging Windows Error Reporting (WER)'s WerFaultSecure.EXE to bypass Protected Process Light (PPL) protections.
+references:
+    - https://github.com/TwoSevenOneT/WSASS
+    - https://www.zerosalarium.com/2025/09/Dumping-LSASS-With-WER-On-Modern-Windows-11.html
+author: Swachchhanda Shrawan Poudel (Nextron Systems)
+date: 2025-09-16
+tags:
+    - attack.credential-access
+    - attack.t1003.001
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_img:
+        Image|endswith: '\wsass.exe'
+    selection_cli:
+        CommandLine|contains: 'WerFaultSecure.exe'
```

Comment:
This overlaps with 1f0b4cac-9c81-41f4-95d0-8475ff46b3e2

Can you please take care of that and perhaps merge this part with the other?